docs/migration-guide/to-16.md

```diff
@@ -45,6 +45,15 @@ async function lint() {
 
 If you use `stylelint.lint()` to lint files, the `code` property will always be `undefined`.
 
+## Changed Node.js API returned `formatters` object
+
+We've changed the `stylelint.formatters` object in the Node.js API so that every formatter is a `Promise` function.
```

Comment:
This might be confusing.
To me it sounds like a breaking change but it's not.
Formatters have just have been thenified while still being backward compatible with existing code.

Reply:
I wouldn't necessarily say there are no problems, but most cases would work because this never changes the `Formatter` function signature.

For example, [`stylelint-formatter-pretty`](https://www.npmjs.com/package/stylelint-formatter-pretty) works:

```console
$ bin/stylelint.mjs scripts/*.css -c scripts/visual-config.json --custom-formatter stylelint-formatter-pretty

  scripts/visual.css:1:1
  ⚠  8:3   Expected custom property name to be kebab-case  custom-property-pattern
  ✖  1:1   Unknown rule unit-case.                         unit-case
  ✖  1:1   Unknown rule string-quotes.                     string-quotes
  ✖  1:9   Expected url("foo.css) to be foo.css"           import-notation
  ✖  4:9   Unexpected invalid hex color #4f                color-no-invalid-hex
  ✖  6:1   Expected .foo.bar to have no more than 1 class  selector-max-class
  ✖  7:10  Unexpected unit px                              unit-disallowed-list

  1 warning
  6 errors
```